sentry_sdk/api.py

```diff
@@ -170,59 +147,37 @@ def push_scope(
     callback=None,  # type: Optional[Callable[[Scope], None]]
 ):
     # type: (...) -> Optional[ContextManager[Scope]]
-    hub = Hub.current
-    if hub is not None:
-        return hub.push_scope(callback)
-    elif callback is None:
-
-        @contextmanager
-        def inner():
-            yield Scope()
-
-        return inner()
-    else:
-        # returned if user provided callback
-        return None
+    return Hub.current.push_scope(callback)
 
 
 @scopemethod  # noqa
 def set_tag(key, value):
     # type: (str, Any) -> None
-    hub = Hub.current
-    if hub is not None:
-        hub.scope.set_tag(key, value)
+    return Hub.current.scope.set_tag(key, value)
```

Comment:
As noted in the PR description / commit message, I intentionally decided to always use the pattern `return Hub.current.method(...)`, even when the return value is `None`. The idea behind it is to avoid accidentally returning `None` when the `Hub` method returns something else (e.g. as part of a behavior change). In other words, I find this easier to maintain than to selectively use or not the `return` keyword.